src/main/java/com/bmc/truesight/meter/plugin/jira/util/Utils.java

```diff
@@ -95,104 +95,40 @@ public static Event eventMeterTSI(final String title, final String message, Stri
-                try {
-                    JsonNode statusFields = filterConfiguration.get(com.bmc.truesight.saas.jira.util.Constants.CONFIG_CONDSTATUSFIELDS_NODE_NAME);
-                    if (statusFields != null) {
-                        List<String> condList = null;
-                        condList = obReader.readValue(statusFields);
-                        configuration.setStatusConditionFields(condList);
-                    }
-                } catch (IOException ex) {
-                    java.util.logging.Logger.getLogger(Utils.class.getName()).log(Level.SEVERE, null, ex);
-
-                }
-                try {
-                    JsonNode priorityFields = filterConfiguration.get(com.bmc.truesight.saas.jira.util.Constants.CONFIG_PRIORITY_CONDITION_FIELDS);
-                    List<String> condList = null;
-                    if (priorityFields != null) {
-                        condList = obReader.readValue(priorityFields);
-                    }
-                    configuration.setPriorityConditionFields(condList);
-                } catch (IOException ex) {
-                    java.util.logging.Logger.getLogger(Utils.class.getName()).log(Level.SEVERE, null, ex);
-                }
-            }
-            // Read the payload details and map to pojo
-            try {
-                JsonNode payloadNode = fieldNode.get(com.bmc.truesight.saas.jira.util.Constants.EVENTDEF_NODE_NAME);
-                String payloadString = mapper.writeValueAsString(payloadNode);
-                TSIEvent event = mapper.readValue(payloadString, TSIEvent.class);
-                template.setEventDefinition(event);
-            } catch (IOException e) {
-                throw new ParsingException(StringUtil.format(com.bmc.truesight.saas.jira.util.Constants.PAYLOAD_PROPERTY_NOT_FOUND, new Object[]{}));
-            }
-
-            // Iterate over the properties and if it starts with '@', put it to
-            // itemValueMap
-            Iterator<Map.Entry<String, JsonNode>> nodes = fieldNode.fields();
-            Map<String, FieldItem> fieldItemMap = new HashMap<>();
-            while (nodes.hasNext()) {
-                Map.Entry<String, JsonNode> entry = (Map.Entry<String, JsonNode>) nodes.next();
-                if (entry.getKey().startsWith(com.bmc.truesight.saas.jira.util.Constants.PLACEHOLDER_START_TOKEN)) {
-                    try {
-                        String placeholderNode = mapper.writeValueAsString(entry.getValue());
-                        FieldItem placeholderDefinition = mapper.readValue(placeholderNode, FieldItem.class);
-                        fieldItemMap.put(entry.getKey(), placeholderDefinition);
-                    } catch (IOException e) {
-                        throw new ParsingException(StringUtil.format(com.bmc.truesight.saas.jira.util.Constants.PAYLOAD_PROPERTY_NOT_FOUND, new Object[]{entry.getKey()}));
+        template.setConfig(configuration);
+        if (config.getApp_id() == null) {
+
```

Comment:
Same here should not be a blank body. could be like `if (config.getApp_id() != null) {`

Reply:
Next build i will change